budget-app/src/components/atoms/Avatar/Avatar.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 import styled from 'styled-components';
-import { IAvatarProps } from '../../../models/index';
+import { IAvatar } from '../../../models/index';
 
 const setImage = (src: any) => `
       background-size: cover;
       background-image: url('${src}')
       `;
 
-const SAvatar = styled.div<IAvatarProps>`
+const SAvatar = styled.div<IAvatar>`
```

Comment:
if u defined the interface as a props interface
so probably not all props you wish to have in the styled div here

better be specific as:
`type SAvatarBoxProps = Pick <IAvatar, "src">`

Reply:
`const SAvatar = styled.div< SAvatarBoxProps >`

so you only pick the specific prop from the props